addons/pos_restaurant/static/tests/tours/pos_restaurant.js

```diff
@@ -106,4 +107,19 @@ odoo.define('pos_reataurant.tour.synchronized_table_management', function (requi
     FloorScreen.check.orderCountSyncedInTableIs('T4', '1');
 
     Tour.register('pos_restaurant_sync_second_login', { test: true, url: '/pos/ui' }, getSteps());
+
+    startSteps();
+
+    ProductScreen.do.confirmOpeningPopup();
+    FloorScreen.do.clickTable("5");
+    ProductScreen.do.clickDisplayedProduct("Coca-Cola");
+    BillScreen.do.clickBillButton();
+    BillScreen.check.isQRCodeNotShown();
```

Comment:
before asserting the non-presence of the qrcode, check first if the bill is shown.